build/root/usr/local/bin/entitle.sh

```diff
@@ -12,9 +16,6 @@ extract_entitlement_key() {
         > "$key"
 }
 
-THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-cd ${THIS_DIR}/../..
-
```

Comment:
the file cannot work without something like this:
```
toolbox/entitlement/test.sh
```

this assumes that we are in the `ci-artifacts` main directory
... but actually, maybe we are, as it's also `$HOME` ...

Could you add something like this:
```

if ! [ -f toolbox/entitlement/test.sh ]; then
  echo "FATAL: entitlement scripts not found in $PWD/toolbox/entitlement/"
  echo "INFO: $0 is intended only for running in the 'OpenShift PSAP CI artifacts' image. (INSIDE_CI_IMAGE=$INSIDE_CI_IMAGE)"
  exit 1
fi

```